src/resources/ethernet_esp32.cc

```diff
@@ -200,7 +203,16 @@ PRIMITIVE(init_esp32) {
   mac_config.smi_mdio_gpio_num = mdio_num;
 
   // TODO(anders): If phy initialization fails, we're leaking this.
```

Comment:
Maybe fix this while you're in here? Might be harder than you would think.

Reply:
Will do it in a separate PR.